_targets.R

```diff
@@ -54,6 +55,27 @@ earliest_date <- "1979-10-01"
 # Change dummy date to force re-build of NWIS SC sites and data download
 dummy_date <- "2021-12-13"
 
+# Define land cover datasets to extract 
+sb_ids_NLCD <- c(
+  # ImperviousnessPct_2011: 
+    '57057a9be4b0d4e2b7571fbb',
+  # Imperviousness100mBufferRipZone:
```

Comment:
Would it clutter things up too much to add the DOI to these comment lines? I wanted to take a quick look at the dataset descriptions but a couple of them were hard for me to find with just the internal sb id. Alternatively, we could flesh out descriptions of land use within the repo README, either with this PR or in a new issue.

Reply:
I see the changes you made to sb_ids_NLCD in response to my comment above. I was actually suggesting to add the DOI's or URLs to the commented line (e.g. "ImperviousnessPct_2011 [https://www.sciencebase.gov/catalog/item/57057a9be4b0d4e2b7571fbb]"), but now I'm curious - do the sbtools functions work with the URL too? If so, that's good to know!

Reply:
Now I can respond 🥳 ! 
Yea, haha I did not get that you meant it as a comment. Basically I added the whole url as a str in a list of str `sb_ids`, and then in the function the sb_id is extracted from the whole url in each item in sb_ids. Easily can revert back to previous approach.

Reply:
> Easily can revert back to previous approach though, shall I do that?

It looks like I missed your reply here. I don't have a strong preference for how the sb_ids are formatted. I guess the `strsplit` adds another step in the function where something could go wrong, but your strsplit/sapply function works well for our sb ids here. If you do decide to revert back, I'd suggest just adding the DOI in the comment since it's a little shorter than the full sciencebase url (e.g. https://doi.org/10.5066/F7765D7V for the Wieczorek attributes)